pymc/logprob/transforms.py

```diff
@@ -469,6 +469,10 @@ def measurable_transform_icdf(op: MeasurableTransform, value, *inputs, **kwargs)
     input_icdf = _icdf_helper(measurable_input, value)
     icdf = op.transform_elemwise.forward(input_icdf, *other_inputs)
 
+    # Fail if transformation is not injective
```

Comment:
Curious: why does `tuple` imply an injective transform here?

Reply:
It was just a choice when I first implemented it. It returns a TensorVariable when it's a 1-to-1 inversion and a tuple with the potential TensorVariables when it's a 1-to-many case. Seemed like a natural way of going about it.

Reply:
Can you add this as a comment? Perhaps something like:
```suggestion
    # Fail if transformation is not injective. A TensorVariable is returned in a 1-to-1 inversion,
    # whereas a tuple is return in a 1-to-many case.
```

Reply:
Sounds good

Reply:
Done